python/tank/descriptor/descriptor_config.py

```diff
@@ -52,6 +52,47 @@ def python_interpreter(self):
         """
         raise NotImplementedError("ConfigDescriptor.python_interpreter is not implemented.")
 
+    @property
+    def core_descriptor(self):
+        """
+        Retrieves the core :class:`Descriptor` object associated with this configuration.
+        """
+        if not self.associated_core_descriptor:
+            return None
+
+        if not hasattr(self, "_cached_core_descriptor"):
+            cache_roots = self._io_descriptor.get_cache_roots()
+            self._cached_core_descriptor = Descriptor.create(
+                # FIXME: We need to pass down a connection here...
```

Comment:
Any idea how to solve this Manne? There is no currently authenticated user, so we can't call create_sg_connection and our descriptor might not even have a sg connection available...